charts/kubewarden-defaults/templates/NOTES.txt

```diff
@@ -25,3 +25,10 @@ by changing their `mode` to `protect`.
 
 For more information check out https://docs.kubewarden.io/quick-start.
 Discover ready to use policies at https://artifacthub.io/packages/search?kind=13.
+
+Kubewarden is applying to become a CNCF incubation project. If you'd like to support us, 
+we'd love to hear from you as one of our adopters. 
+
+Adopters can be public or private. 
+
+Feel free to reach out at cncf-kubewarden-maintainers@lists.cncf.io to get involved.
```

Comment:
Maybe add a link to the adopter documentation in the community repository. 
https://github.com/kubewarden/community/blob/main/ADOPTERS.md

Reply:
good point, I've done it now